compiler/rustc_feature/src/active.rs

```diff
@@ -336,6 +336,8 @@ declare_features! (
     (active, closure_track_caller, "1.57.0", Some(87417), None),
     /// Allows to use the `#[cmse_nonsecure_entry]` attribute.
     (active, cmse_nonsecure_entry, "1.48.0", Some(75835), None),
+    /// Allows use of the `#[collapse_debuginfo]` attribute.
+    (active, collapse_debuginfo, "1.62.0", None, None),
```

Comment:
The version number should be current nightly, I think. 
See https://rustc-dev-guide.rust-lang.org/feature-gates.html#adding-a-feature-gate

Reply:
Updated, it was just outdated because of when the PR was opened..